src/components/BookingForm.vue

```diff
@@ -31,26 +37,38 @@
 
 <script>
 import BadContainedButton from "@/components/BadContainedButton";
+import BadTextInput from "@/components/BadTextInput";
+
 export default {
   name: "BookingForm",
   data() {
     return {
       officeId: "",
-      bookingdate: "",
-      emailaddress: ""
+      bookingDate: "",
+      emailAddress: ""
     };
   },
   methods: {
+    changeOfficeId(value) {
+      this.officeId = value;
+    },
+    changeBookingDate(value) {
+      this.bookingDate = value;
+    },
+    changeEmail(value) {
+      this.emailAddress = value;
+    },
     submitBooking() {
       this.$store.dispatch("book", {
-        officeId: this.officeId,
-        bookingdate: this.bookingdate,
-        emailaddress: this.emailaddress
+        OfficeId: this.officeId,
```

Comment:
There are still uncertainties of what BE expect. We can update it once we figure for sure.

https://broadsign.slack.com/archives/C01AU5P352L/p1607719055005400

@devtxdotnet can you please confirm the params?